docs/source/markdown/podman-container-clone.1.md.in

```diff
@@ -92,21 +92,26 @@ When set to true, this flag runs the newly created container after the
 clone process has completed, this specifies a detached running mode.
 
 ## EXAMPLES
+
+Clone current container into a new container:
```

Comment:
I wondered what "current" means.  Other man pages use "specified" instead which I find clearer and consistent.

Reply:
Still there